index.html

```diff
@@ -1,8 +1,9 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <title>Yandex Messengerff</title>
+  <title>Yandex Messenger</title>
   <meta charset="utf-8">
+  <link rel="shortcut icon" type="image/png" href="https://praktikum.s3.yandex.net/praktikum/v0.187.13-1635776906/static/favicon.ico"/>
 </head>
 <body>
   <div id="page-content"></div>
```

Comment:
поскольку ты используешь HtmlWebpackPlugin, то отсюда стоит удалить строку `<script src="./src/pages/index.ts"></script>`. при сборке этот плагин сам добавит нужный скрипт в документ. сейчас вот так выглядит этот html файлик
<img width="539" alt="Screenshot 2021-11-10 at 09 02 54" src="https://user-images.githubusercontent.com/33020166/141058851-9dbb33b3-0faa-4892-82b2-6771b1cb39e2.png">

то есть в рантайме у меня подключается `.ts` файл, о таком расширении браузер ничего не знает, поэтому в консоле ошибка `Refused to execute script from 'https://sheltered-inlet-27428.herokuapp.com/52acc7b36268a2f65a9f.ts' because its MIME type ('video/mp2t') is not executable.`